web/html/app_middleware.go

```diff
@@ -0,0 +1,62 @@
+package html
+
+import (
+	"context"
+	"net/http"
+	"path"
+	"strings"
+
+	"github.com/madappgang/identifo/model"
+	"github.com/urfave/negroni"
+)
+
+const (
+	//FormKeyAppID form key to keep application ID
+	FormKeyAppID = "appId"
+)
+
+// AppID gets app id from the request body
+func (ar *Router) AppID() negroni.HandlerFunc {
+	errorPath := path.Join(ar.PathPrefix, "/misconfiguration")
```

Comment:
I would give the better name for this route

Reply:
Like what? Because I actually didn't get the point why you don't like this name.